download.html

```diff
@@ -152,7 +152,7 @@ <h3><b>Instructions</b></h3>
                     <span class="anchor" id="link"></span>
                     <h3><b>Download Links</b></h3>
                     <p> 
-                        Download link:  <a href="https://www.kaggle.com/datasets/hsj576/cloud-robotics" target="_blank">Cloud-Robotics</a></br>
+                        The dataset is currently in the testing phase. If you want to get the beta version of the dataset, please email sjhu21@m.fudan.edu.cn.
```

Comment:
Polished version: "This dataset belongs to the perception module of the cloud-robotic benchmarking project which is in its beta stage and will be available shortly. Before the official release of the whole project, one can request a copy of this dataset by emailing Mr. Shijing via sjhu21@m.fudan.edu.cn."